doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -198,7 +200,9 @@ With the Dataverse-specific config in place, you can now start Solr and create t
         bin/solr start
         bin/solr create_core -c collection1 -d server/solr/collection1/conf/
 	
-Please note: Solr will warn about needing to increase the number of file descriptors and max processes in a production environment but will still run with defaults. We have increased these values to the recommended levels by adding ulimit -n 65000 to the init script and adding solr soft nproc 65000 to /etc/security/limits.conf. If anyone knows of a better way to do this, especially for CentOS 7 boxes, please let us know.
+Please note: Solr will warn about needing to increase the number of file descriptors and max processes in a production environment but will still run with defaults. We have increased these values to the recommended levels by adding ulimit -n 65000 to the init script and adding solr soft nproc 65000 to /etc/security/limits.conf. On operating systems which use systemd such as RHEL or CentOS 7, you may add a line like LimitNOFILE=65000 to the systemd unit file, or adjust the limits on a running process using the prlimit tool:
+
+	$ sudo prlimit -p pid -n 65000:65000
```

Comment:
Same here as my previous comment. This example command is not formatted like other examples on the page (see Line 87), missing a `#` character to format correctly.

Reply:
Also, not sure if it required, but inline command/setting/code examples can/should be formatted using double backquote characters (``) which could apply to ulimit -n 65000, solr soft nproc 65000, /etc/security/limits.conf, and LimitNOFILE=65000.